src/federation/transaction.rs

```diff
@@ -230,3 +221,10 @@ pub struct DebugTransaction {
     inputs: Vec<String>,
     outputs: Vec<String>,
 }
+
+#[derive(Debug, Clone, FromRow)]
+pub struct HistogramEntry {
+    date: String,
+    count: i64,
+    amount: i64,
```

Comment:
There are a bunch of fields that I made `i*` instead of `u*` to avoid duplicating a lot of structs. It's a shame that databases seem to insist on signed integers. I'll have to clean up types a bit I guess.